lib/ddtrace/contrib/dalli/quantize.rb

```diff
@@ -8,10 +8,13 @@ module Quantize
         module_function
 
         def format_command(operation, args)
+          placeholder = "#{operation} BLOB (OMITTED)"
           command = [operation, *args].join(' ').strip
+          command = Utils.utf8_encode(command, binary: true, placeholder: placeholder)
           Utils.truncate(command, MAX_CMD_LENGTH)
-        rescue ::Encoding::CompatibilityError
-          "#{operation} BLOB (OMITTED)"
+        rescue => e
+          Tracer.log.error("Error sanitizing Dalli operation: #{e}")
```

Comment:
can you set this to `debug`? otherwise we may hit a critical path with a lot of log entries.